source/PCM_API.c

```diff
@@ -0,0 +1,216 @@
+/** 
+ * @file        PCM_API.c
+ *
+ * @brief       PCM_API contains the PLL basic functions
+ *
+ * This Module implements the PLL basis functionalities. The Module contains
+ * functions to set the PLL output frequencys.
+ * 
+ * @author      Florian Heidecker
+ * @date        26.11.2015 - initial version
+ * 
+ * @version     0.1
+ *
+ */
+
+#include <xc.h>
+
+#include "log.h"
+#include "PCM_API.h"
+#include "SPI_API.h"
+
+#define pcm_channel_dummy       0x01 //!!!!!!!!!!!!!!!!!!!!
+
+/** avalible Register 16 - 23 */
+typedef enum {	PCM_register_16_w = 0x10,   PCM_register_16_r = 0x90,
+                PCM_register_17_w = 0x11,   PCM_register_17_r = 0x91,
+                PCM_register_18_w = 0x12,   PCM_register_18_r = 0x92,
+				PCM_register_19_w = 0x13,   PCM_register_19_r = 0x93,
+				PCM_register_20_w = 0x14,   PCM_register_20_r = 0x94,
+                PCM_register_21_w = 0x15,   PCM_register_21_r = 0x95,
+                                            PCM_register_22_r = 0x96,
+                                            PCM_register_23_r = 0x97} pcm_register_t;
+
+void pcm_init(void){
+    PCM_LOG("PCM init\n");
+    
+    // todo !!!!!!!!!!
+}
+
+uint8_t send_receive(uint8_t address, uint8_t data){
+    uint8_t data_send[2] = {address, data};
+    uint8_t data_receive[2] = {0, 0};
+    
```

Comment:
Tip:
Format Adress and Data as hex (0x%x)
I think its easier to interpret the log.
But its up to you.